lib/state/persistor.dart

```diff
@@ -36,12 +36,12 @@ class AppPersistor extends Persistor<AppState> {
   }
 
   String stateEncode(AppState st) {
-    final Map<String, dynamic> json = st.toJson();
+    final json = st.toJson();
```

Comment:
I know dart is intelligent enough to know the object definition of this property but may I know if you have other reason for removing the prior object declaration? 🤔 🙂

Reply:
This was brought up by the lints checker

Reply:
aight. thanks 🙂